test/conformance/walk.go

```diff
@@ -72,6 +72,8 @@ type conformanceData struct {
 	TestName string
 	// Extracted from the "Description:" comment before the test
 	Description string
+	// Version when this test is added or modified ex: 1.12, 1.13
```

Comment:
Probably worth being consistent and documenting the extraction prefix.

Reply:
Done